daemon/daemon.go

```diff
@@ -743,12 +760,14 @@ func NewDaemon(config *Config, registryService *registry.Service) (daemon *Daemo
 	return d, nil
 }
 
+// Shutdown stops the daemon.
 func (daemon *Daemon) Shutdown() error {
 	if daemon.containers != nil {
 		group := sync.WaitGroup{}
 		logrus.Debug("starting clean shutdown of all containers...")
 		for _, container := range daemon.List() {
-			c := container
+			var c *Container
+			c = container
```

Comment:
whoops, relic of my understanding embedding. Definitely putting that back. Thanks.